projects/mci/src/regen_braking.c

```diff
@@ -16,11 +16,11 @@ static StatusCode prv_regen_braking_callback(const CanMessage *msg, void *contex
 
 StatusCode regen_braking_init(void) {
   // Default to enabled
-  s_regen_braking_state = REGEN_ENABLED;
+  s_regen_braking_state = 1;
```

Comment:
FYI you can use `true` and `false` instead of 0 and 1 if you have `<stdbool.h>` included

Reply:
done